conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,298 @@
+# This is CORTX GConf (Config Store) which serves as input 
+# to all the CORTX Services
+# Location: /etc/cortx/cluster.conf 
+
+cortx:
+  external:
+    consul:
+      admin: admin
+      endpoints:
+      - http://consul-server.cortx-cluster.com:80
+      secret: !!binary |
+        MG12NU5BVjBVeWpBSWJuSV9uVjV5UzUxN3lTSVNkNUk1YkRzbjd2QkE9PQ==
+    kafka:
+      admin: admin
+      endpoints:
+      - kafka-server1.cortx-cluster.com
+      - kafka-server2.cortx-cluster.com
+      secret: !!binary |
+        TmRPNThMSm5GOVl5RHgwTVFfaTRJZ3k3YUUwMldmNkdxRE1KYVIxc0E9PQ==
+    openldap:
```

Comment:
for now this is ok, but by end of Sprint 62 we should remove this. Openldap will not be required

Reply:
@ujjwalpl should I remove "openldap" right away or  to be done as part of another PR later in the end of sprint 62

Reply:
Yes, if this patch is getting into RGW, we can remove ldap

Reply:
done

Reply:
done